wiki/docs/toolchain.md

```diff
@@ -0,0 +1,14 @@
+Toolchain
+---
+
+To build Mimiker you will need a custom toolchain we use. It contains cross
+compiler (`gcc`), linker (`binutils`) and debugger (`gdb`). We prepared prebuild
```

Comment:
Is it still true after transition to llvm?

Reply:
It's still valid. For some period of time GNU toolchain will be second class citizen before support for it is removed.

Reply:
Huh? When I'm reading this paragraph I have the following thoughts:
  * `gcc`, `ld.bfd` and `gdb` are *required* to build & run Mimiker.
  * I can run it only on x86-64 not specified version of Debian using `qemu` with unknown patches.

First is not true. You can run Mimiker without `gdb` - probably it's not what you want to achieve... but you can.
`gcc` is *not* required - since we support `llvm` - personally I don't have custom gnu toolchain and I'm using only default clang which is enough to build.
We should mark that all patches to qemu (and other toys that we are using) are open-source and they are located in toolchain directory and end user can build everything using our build system.